content/pages/pricing.html

```diff
@@ -219,8 +219,8 @@ <h3 class="ui small header">
     {% call pricing.feature_table() %}
       {{ pricing.feature_row("Team management", "Control organization access with team permissions", "fa-users", on_community=False) }}
       {{ pricing.feature_row("Email support", "Get help with your projects and documentation", "fa-message-question", "Best effort", "2 business days", "2 business days", ("1 business day", "olive")) }}
+      {{ pricing.feature_row("Ad free", "Documentation without ads", "fa-ad", on_community=False) }}
```

Comment:
I'd also find a more specific phrase to use here. I think the reason this feels less clear is that this changing this to "with"/"without" removes the verb from the phrase.

Reply:
```suggestion
      {{ pricing.feature_row("Ad free", "Documentation doesn't include ads", "fa-ad", on_community=False) }}
```

Perhaps just inverting the original?

Reply:
The sounds clearer. We've also used "documentation is ad-free" on our marketing or docs pages too.